### DISCUSSION TITLE
Axios REST client - find() does not pass headers, all others (get, create, removed) does

### DISCUSSION THREAD

**Discussion by SharpBCD** [feathersjs/feathers#2798] [Q&A] [answered]
REST api, using axios. Configured like this:
```
    const axios = require("axios");
   const restClient = rest(serverPaths.serverPath.slice(0, -1));
  restApi.configure(restClient.axios(axios.create({
		  headers: {accessToken: '--change---me'}
	  })));
```
the header part is just to test and indeed, this header is passed always.

The request like this:
```
const sr = await this.$restApi.service('users').find({}, {
					headers: {
						accesstoken: '-this is a new token-'
					}
				});
```
Does not pass the header. Tested in browser. 
Replace `find` with anything else and it works.

**Comment by daffl** [comment, marked as answer]
The correct method call is

```js
const sr = await this.$restApi.service('users').find({
  headers: {
    accesstoken: '-this is a new token-'
  }
})
```

You passed an empty object as the first argument which already is the `params` object for `find` calls.

**Reply by SharpBCD** [reply]
uhh.. right. So for `find` is {query, params}. Now it's working.
